### ISSUE TITLE
How to compile for linux?

### ISSUE THREAD

**Issue by catfoodnaow** [Yakibomb/LAZYSHELL-UPDATED#2]
I don't see any make or .config files anywhere, how do I compile this thing?

**Comment by Proxieru** [comment]
Coming back to revisit this after actually learning some stuff, you cannot compile this for linux in it's current state because it targets .NET 4.8 and as far as I am aware the SDK for .NET 4.8 is not installable on linux and thus it cannot be built for linux. But even if it could it uses WinForms which only runs on windows (this can be worked around if a .exe is run with mono) and it also relies on a .dll that is windows only and closed source so it cannot be compiled for linux as a .so. I've tried to get this working on linux as a personal project but I didn't want to bother rewriting or writing a .so that could do the purpose of the windows only .dll needed.

I know the original poster most likely will never see this but I wanted to rewrite my response to actually provide a answer because
now I know more than I did when I originally posted this reply to this issue.

I hope this helps.

**Comment by pidgezero-one** [comment]
I got this working well enough on wine.

**Comment by Proxieru** [comment]
Yes that is mainly how I used it. but from my experience using it under wine, some text boxes don’t work properly such as in the monster editor when editing monster names.
